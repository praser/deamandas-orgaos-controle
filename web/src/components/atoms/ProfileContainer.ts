import styled from 'styled-components';

const ProfileContainer = styled.div`
  display: flex;
  align-items: center;
`;

export default ProfileContainer;
