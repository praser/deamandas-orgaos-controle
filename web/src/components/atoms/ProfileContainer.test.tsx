import React from 'react';
import { render, screen } from '@testing-library/react';
import ProfileContainer from './ProfileContainer';

describe('ProfileContainer atomic component', () => {
  let profileContainer: HTMLElement;

  beforeEach(() => {
    render(<ProfileContainer data-testid="profile-container" />);
    profileContainer = screen.getByTestId('profile-container');
  });

  it('is expected to match the snapshot', () => {
    expect(profileContainer).toMatchSnapshot();
  });
});
