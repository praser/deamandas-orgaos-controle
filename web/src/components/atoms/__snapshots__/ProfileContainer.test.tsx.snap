// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileContainer atomic component is expected to match the snapshot 1`] = `
<div
  class="sc-bdvvaa fcGAIO"
  data-testid="profile-container"
/>
`;
